### PR TITLE
Add initial fun facts to nandanthakkar progile

### PR DESCRIPTION
**Pull request by nandanthakkar**

### CODE REVIEW

**Comment by nandanthakkar** [comment]
:100:

**Comment by addster1018** [comment]
keeping it 100

File: nandanthakkar-fun.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## java
```
Comment (nandanthakkar) [inline nandanthakkar-fun.md:1]: ha hahahahaha

**Comment by addster1018** [comment]
:+1

**Pull request merged**